README.md

```diff
@@ -57,31 +61,32 @@ For an example use-case, let's make a simple class that multiplies two numbers,
 ```python
 @loggo
 class Multiplier(object):
```

Comment:
Not really in the scope, but if you wish, remove the `object` inheritance. We dont do python2...

Reply:
fixed